src/main/java/controller/AttendanceController.java

```diff
@@ -0,0 +1,125 @@
+                printRiskOfExpulsion(attendanceSheets);
+                continue;
+            }
+
+            if (select.equals("Q")) {
+                return;
+            }
+        }
+    }
+
+    private void printRiskOfExpulsion(AttendanceSheets attendanceSheets) {
+        List<String> allNames = attendanceSheets.findAllNames();
+
+        OutputView.printRiskOfExpulsionBanner();
+
+        for (String name : allNames) {
+            int lateCount = attendanceSheets.calculateLateCountBy(name);
+            int absentCount = attendanceSheets.calculateAbsentCount(name, today);
+
+            AbsentPolicy absentPolicy = AbsentPolicy.calculateAbsentPolicy(absentCount, lateCount);
+            if (AbsentPolicy.isRiskOfExpulsion(absentPolicy)) {
+                OutputView.printRiskOfExpulsion(name, lateCount, absentCount, absentPolicy);
+            }
+        }
+        System.out.print(System.lineSeparator());
+    }
+
+    private void printAttendanceSheetsByCrew(AttendanceSheets attendanceSheets) {
+        String nickname = inputView.inputNickname();
+        List<AttendanceSheet> attendancesByNickname = attendanceSheets.findAttendanceByNickname(nickname);
+        OutputView.printAttendanceSheetsByCrew(nickname, attendancesByNickname, attendanceSheets, today);
+    }
+
+    private void updateAttendance(AttendanceSheets attendanceSheets) {
+        String nickname = inputView.inputUpdateNickname();
+        int day = Integer.parseInt(inputView.inputUpdateDate());
+
+        AttendanceSheet attendanceSheet = attendanceSheets.findAttendanceSheetByNicknameAndDay(nickname,
+                day);
+
+        String time = inputView.inputUpdateTime();
+        int hour = Integer.parseInt(time.split(":")[0]);
+        int minute = Integer.parseInt(time.split(":")[1]);
+
+        AttendanceDateTime attendanceDateTime = attendanceSheet.getAttendanceDateTime();
+        AttendanceTime beforeAttendanceTime = new AttendanceTime(
+                LocalTime.of(
+                        attendanceDateTime.getAttendanceTime().getHour(),
+                        attendanceDateTime.getAttendanceTime().getMinute())
+        );
+
+        AttendanceState beforeState = attendanceDateTime.check();
+        attendanceDateTime.update(LocalTime.of(hour, minute));
+        AttendanceState afterState = attendanceDateTime.check();
+
+        OutputView.printUpdateInformation(beforeAttendanceTime, attendanceDateTime, beforeState, afterState);
+    }
+
+    private void attend(AttendanceSheets attendanceSheets) {
+        String nickname = inputView.inputNickname();
+        String time = inputView.inputTime();
+        int hour = Integer.parseInt(time.split(":")[0]);
+        int minute = Integer.parseInt(time.split(":")[1]);
+
+        AttendanceDateTime attendanceDateTime = AttendanceDateTime.from(
+                LocalDateTime.of(today.getYear(), today.getMonth(), today.getDayOfMonth(), hour,
+                        minute));
```

Comment:
```suggestion
                LocalDateTime.of(today, LocalTime.of(hour, minute));
```

이렇게 수정하는건 어떨까요? 가독성이 더 좋을 것 같습니다!

Reply:
이 부분은 확실히 고친 것이 가독성이 더 좋네요!! 바꾸었습니다.